package.json

```diff
@@ -6,7 +6,7 @@
     "@ant-design/icons": "5.0.1",
     "@chakra-ui/react": "^2.8.2",
     "@descope/node-sdk": "1.6.8",
-    "@descope/react-sdk": "2.0.35",
+    "@descope/react-sdk": "^2.0.35",
```

Comment:
This is a version from 5 months ago, no way it can work, you should use the correct version here.
![image](https://github.com/user-attachments/assets/b973ac19-40ca-44f3-b2e2-76c949f32ce2)

Reply:
What about this comment?

Reply:
do you suggest to lock the sdk version? because the ^ works

Reply:
You should use the relevant version